src/components/my-criteria/my-criteria.tsx

```diff
@@ -0,0 +1,153 @@
+
+    const interval = setInterval(async () => {
+      const isStillInClipboard = await isContentInClipboard(copiedContent);
+      if (!isStillInClipboard) setCopiedContent(null);
+    }, 3000);
+
+    return () => clearInterval(interval);
+  }, [copiedContent]);
+
+  const isContentInClipboard = async (content: string) => {
+    try {
+      const clipboardText = await navigator.clipboard.readText();
+      return clipboardText === content;
+    } catch (err) {
+      console.error('Failed to read clipboard content:', err);
+      return false;
+    }
+  };
+
+  return (
+    <div className="MagentaA11y__my-criteria">
+      <div className="MagentaA11y__my-criteria__heading-wrapper">
+        <h1 className="MagentaA11y__my-criteria__heading">My Criteria</h1>
+        <p className="MagentaA11y__my-criteria__headline">
+          Lorem ipsum about this
+        </p>
+      </div>
+      <Divider orientation={OrientationEnum.HORIZONTAL} />
+      {savedCriteria.length > 0 && (
+        <div className="MagentaA11y__my-criteria__actions">
+          <Chips
+            variant={ChipType.BUTTON}
+            chips={criteriaChips}
+            onDelete={handleDelete}
+            size={ChipSize.SMALL}
+            legend="Saved Criteria"
+            ref={chipsContainerRef}
+          />
+          {criteria.length > 1 && (
+            <div className="w-100">
+              <Button
+                onClick={() =>
+                  clearCriteria(uniqueLabels[activeTab] as CriteriaType)
+                }
+                type={ButtonType.button}
+                variant={ButtonVariant.tertiary}
+                size={ButtonSize.large}
+                label={'Remove All'}
+                id="clear-all-btn"
+              />
+              <Divider orientation={OrientationEnum.HORIZONTAL} />
+            </div>
+          )}
+          <div className="MagentaA11y__my-criteria__tab-container">
+            <md-tabs
+              ref={tabsRef}
+              aria-label="Content options for syntax"
+              role="tablist">
+              {uniqueLabels.map((label, index) => (
+                <md-primary-tab
+                  key={label}
+                  aria-selected={activeTab === index ? 'true' : 'false'}
+                  id={`${label.toLowerCase().replace(/\s+/g, '-')}-tab`}
+                  role="tab"
+                  onClick={() => setActiveTab(index)}>
+                  {label}
+                </md-primary-tab>
+              ))}
+            </md-tabs>
+
+            <Button
+              onClick={copyCriteria}
+              type={ButtonType.button}
+              variant={ButtonVariant.primary}
+              size={ButtonSize.large}
+              label={copiedContent ? 'Copied!' : 'Copy Criteria'}
+              decoration={copiedContent ? Icons.checkmark : Icons.copyFilled}
+              id="copy-criteria"
+            />
+          </div>
+        </div>
+      )}
+
+      <MarkdownContent tabs={criteria} />
```

Comment:
for anywhere tabs are present, we need a role of "tabpanel" where the tab content will be displayed: https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/

Reply:
ARC-490 has to do with this issue

Reply:
also can we do the changes to the tablist recommended in ARC-489 too?